src/config.py

```diff
@@ -185,17 +218,31 @@ def server_properties(self) -> List[str]:
         Returns:
             List of properties to be set
         """
-        return (
-            [
-                f"offsets.retention.minutes={self.charm.config['offsets-retention-minutes']}",
-                f"log.retention.hours={self.charm.config['log-retention-hours']}",
-                f"auto.create.topics={self.charm.config['auto-create-topics']}",
-                f"super.users={self.super_users}",
+        properties = [
+            f"offsets.retention.minutes={self.charm.config['offsets-retention-minutes']}",
+            f"log.retention.hours={self.charm.config['log-retention-hours']}",
+            f"auto.create.topics={self.charm.config['auto-create-topics']}",
+            f"super.users={self.super_users}",
+        ] + self.default_replication_properties + self.auth_properties + DEFAULT_CONFIG_OPTIONS.split("\n")
```

Comment:
Biggest nit in the world. Can these go back to separate lines?
Helps when commenting them out during debugging.

Reply:
Sure. I think it was the overzealous linter changing it at some point?